docs/1.0.0/install_macos.md

```diff
@@ -0,0 +1,230 @@
+        --network-id 33
+        --eth-rpc-endpoint $RSK_NODE_URL
+        --environment-type development
+        --tokennetwork-registry-contract-address $TOKENNETWORK_REGISTRY_CONTRACT_ADDRESS
+        --secret-registry-contract-address $SECRET_REGISTRY_CONTRACT_ADDRESS
+        --endpoint-registry-contract-address $ENDPOINT_REGISTRY_CONTRACT_ADDRESS
+        --no-sync-check
+        --api-address 127.0.0.1:5001
+        --rnsdomain $YOUR_RNS_DOMAIN
+        --discoverable # if this flag is present, then your node will be registered on Lumino Explorer
+        --hub-mode # if this flag is present, then your node will run in HUB mode
+        --transport # transport mode
+    ```
+    
+    | FIELD                                     | DESCRIPTION                                                                                                                             |
+    |-------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------|
+    | `$KEYSTORE_PATH`                          | The path to your keystore folder.                                                                                                       |
+    | `network-id`                              | The blockchain network ID you're connecting to. This must match the RSK node you're connecting to. `33` is regtest.                     |
+    | `$RSK_NODE_URL`                           | URL of the RSK node to connect to (`http://URL:PORT`). If you're running a local node, this will typically be `http://localhost:4444/`. |
+    | `environment-type`                        | You will need this set to `development` in order to use custom blockchain and contract addresses.                                       |
+    | `$TOKENNETWORK_REGISTRY_CONTRACT_ADDRESS` | Address for the token registry contract deployed (view contracts table).                                                                |
+    | `$SECRET_REGISTRY_CONTRACT_ADDRESS`       | Address for the secret registry contract deployed (view contracts table).                                                               |
+    | `$ENDPOINT_REGISTRY_CONTRACT_ADDRESS`     | Address for the endpoint registry contract deployed (view contracts table).                                                             |
+    | `no-sync-check`                           | This will allow you to bypass checking that the node is synchronized against etherscan.                                                 |
+    | `$YOUR_RNS_DOMAIN`                        | You can supply the RNS address associated with your RSK node address, e.g. `--rnsdomain=lumino.rsk.co`                                 |
+    | `transport`                               | Transport mode for Lumino, rif-comms and matrix are supported. Defaults to rif-comms, e.g. `--transport=matrix`                                 |
```

Comment:
add thats defaults to rif comms using gprc port 5013

add missing --grpc-endpoint flag

Reply:
i would add something like:

```markdown
| `grpc-endpoint` | The communication endpoint for the RIF Comms node used for transport purposes. Note that the `transport` flag must be set to `rif-comms` (explicitly or by default) for this parameter to be used. Defaults to `"localhost:5013"`. |
```

Reply:
done